opbeans/pom.xml

```diff
@@ -39,9 +38,9 @@
 			<version>1.28.4</version>
 		</dependency>
 		<dependency>
-			<groupId>co.elastic.apm</groupId>
-			<artifactId>apm-opentracing</artifactId>
-			<version>1.3.0</version>
+			<groupId>io.opentelemetry</groupId>
+			<artifactId>opentelemetry-api</artifactId>
+			<version>1.9.1</version>
```

Comment:
Seems like the [latest](https://github.com/open-telemetry/opentelemetry-java) available version is v1.11.0. Should we take this into account ?